src/index.html

```diff
@@ -5,8 +5,482 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__item-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__item-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__item-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              class="header__item-link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__item-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__item-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__item-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__item-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__cards-first_row">
```

Comment:
you don't need an additional container for each row. it would be better to add a `max-width` to a main container

Reply:
I understood correctly that it is only necessary to add
`<main class="main max-width">?`

Reply:
No, I guess the comment was about [max-width CSS property ](https://developer.mozilla.org/en-US/docs/Web/CSS/max-width). 
```suggestion
    <main class="main">
```